### PR TITLE
Mount doc

### PR DESCRIPTION
**Pull request by Blopgrop**
Thank you for submitting a pull request (PR)! Before hitting the submit button, we would be really glad if you check all the following points. Even if you can't check all the boxes below, do not hesitate to go ahead with the PR and ask for help there.

Please also refer to the doc on [contributing](https://lensless.readthedocs.io/en/latest/contributing.html) for more details. 

- [x] Have you run the tests by running `pytest` from the repository root? See the [documentation](https://lensless.readthedocs.io/en/latest/contributing.html#unit-tests) for more details.
- [x] Do you follow the code formatting for this project? This can be done by setting up pre-commit hooks as described in the [documentation](https://lensless.readthedocs.io/en/latest/contributing.html#coding-style).
- [ ] Is there a unit test for the proposed code modification? If the PR addresses an issue, the test should make sure the issue is fixed.
- [x] Are there docstrings? Do they follow the [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard) style?
- [x] Have you checked that the doc builds properly and that any new RST file has been added to the repo? How to do that is covered in the [documentation](https://lensless.readthedocs.io/en/latest/contributing.html#building-documentation).
- [x] Did you document the proposed change(s) in the CHANGELOG file? It should go under "Unreleased".

Happy PR :smiley:

### CODE REVIEW

**Comment by Blopgrop** [comment]
Need to do unit test for mask adapter creation

File: docs/source/fabrication.rst [hunk 1]
```diff
@@ -7,6 +7,14 @@
       :alt: Mount components.
       :align: center
 
+ 
+   Note that the most recent version of the mount looks like this, with the addition of stoppers, 
+   to prevent the mask from scratching the Pi Camera.
+   This new version of the mount can be found `here <https://drive.switch.ch/index.php/s/QYH9vSqPd21DIHQ>`_
+
+   .. image:: mount_V4.png
```
Comment (ebezzam) [inline docs/source/fabrication.rst:15]: do you need to add this image?

File: docs/source/fabrication.rst [hunk 2]
```diff
@@ -22,6 +30,14 @@
       :members:
       :special-members: __init__
 
+   Because newer versions of the masks' size are smaller, the following adapter enables
+   them to be used with the current mounts design.
+
+   .. image:: mask_adapter.png
```
Comment (ebezzam) [inline docs/source/fabrication.rst:36]: do you need to add this image?

File: docs/source/fabrication.rst [hunk 3]
```diff
@@ -7,6 +7,14 @@
       :alt: Mount components.
       :align: center
 
+ 
+   Note that the most recent version of the mount looks like this, with the addition of stoppers, 
+   to prevent the mask from scratching the Pi Camera.
+   This new version of the mount can be found `here <https://drive.switch.ch/index.php/s/QYH9vSqPd21DIHQ>`_
```
Comment (ebezzam) [inline docs/source/fabrication.rst:13]: have you uploaded files here?

**Pull request merged**